data/topic-filter.go

```diff
@@ -0,0 +1,222 @@
+	}
+
+	// EthnicGroupNationalIdentityAndReligion - Search information specific for ethnic groups, national identity and religion
+	EthnicGroupNationalIdentityAndReligion = TopicContentType{
+		LocaliseKeyName: "EthnicGroupNationalIdentityAndReligion",
+		Group:           "ethnic_group_national_identity_and_religion",
+		Types:           []string{"ethnic_group_national_identity_and_religion"},
+		ShowInWebUI:     true,
+	}
+
+	// HealthDisabilityAndUnpaidCare - Search information specific for health, disabilities and unpaid care
+	HealthDisabilityAndUnpaidCare = TopicContentType{
+		LocaliseKeyName: "HealthDisabilityAndUnpaidCare",
+		Group:           "health_disability_and_unpaid_care",
+		Types:           []string{"health_disability_and_unpaid_care"},
+		ShowInWebUI:     true,
+	}
+
+	// Housing - Search information specific for Housing
+	Housing = TopicContentType{
+		LocaliseKeyName: "Housing",
+		Group:           "housing",
+		Types:           []string{"housing"},
+		ShowInWebUI:     true,
+	}
+
+	// LabourMarketAndTravelToWork - Search information specific for labour market and travel to work
+	LabourMarketAndTravelToWork = TopicContentType{
+		LocaliseKeyName: "LabourMarketAndTravelToWork",
+		Group:           "labour_market_and_travel_to_work",
+		Types:           []string{"labour_market_and_travel_to_work"},
+		ShowInWebUI:     true,
+	}
+
+	// SexualOrientationAndGenderIdentity - Search information specific for sexual orientation and gender identity
+	SexualOrientationAndGenderIdentity = TopicContentType{
+		LocaliseKeyName: "SexualOrientationAndGenderIdentity",
+		Group:           "sexual_orientation_and_gender_identity",
+		Types:           []string{"sexual_orientation_and_gender_identity"},
+		ShowInWebUI:     true,
+	}
+
+	// Veterans - Search information specific for Veterans
+	Veterans = TopicContentType{
+		LocaliseKeyName: "Veteran",
+		Group:           "veterans",
+		Types:           []string{"veterans"},
+		ShowInWebUI:     true,
+	}
+
+	// topicFilterOptions contains all the possible filter available on the search page
+	topicFilterOptions = map[string]TopicContentType{
+		DemographyAndMigration.Group:                 DemographyAndMigration,
+		Education.Group:                              Education,
+		EthnicGroupNationalIdentityAndReligion.Group: EthnicGroupNationalIdentityAndReligion,
+		HealthDisabilityAndUnpaidCare.Group:          HealthDisabilityAndUnpaidCare,
+		Housing.Group:                                Housing,
+		LabourMarketAndTravelToWork.Group:            LabourMarketAndTravelToWork,
+		SexualOrientationAndGenderIdentity.Group:     SexualOrientationAndGenderIdentity,
+		Veterans.Group:                               Veterans,
+	}
+)
+
+// reviewFilter retrieves filters from query, checks if they are one of the filter options, and updates validatedQueryParams
+func reviewTopicFilters(ctx context.Context, urlQuery url.Values, validatedQueryParams *SearchURLParams) error {
+	topicFiltersQuery := urlQuery["topic"]
```

Comment:
Shouldn't the value of query parameter be `topics` instead of `topic`? We can have more than one comma separated list of topics

Reply:
So we don't want the same feature in the url as filter.

The url should look like: `ons.gov.uk/search?q=xxx&topics=census-age,census-health-and-wellbeing` **not** `ons.gov.uk/search?q=xxx&topic=census-age&topic=census-health-and-wellbeing`

Therefore the `topicFiltersQuery` is of type string and probably renamed to just `topicFilters`

Reply:
As it is a string, you will then need to split the string by comma into a new type []string

like so:

```
    topicFilters := urlQuery["topics"]
    topics := strings.Split(topicFilters, ",")

    for _, topic := topics {
        ...
```